app/views/sdg/goals/help.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide(:title) { t("sdg.goals.help.title") } %>
+
+<% content_for :canonical do %>
+  <%= render "shared/canonical", href: sdg_help_url %>
+<% end %>
+
+<section class="row sdg-help-content">
+  <%= back_link_to help_path %>
+  <h1><%= t("sdg.goals.help.title") %></h1>
+  <p><%= t("sdg.goals.help.description") %></p>
+  <ul class="tabs" id="help_tabs" data-deep-link="true" data-tabs>
+    <% @goals.each do |goal| %>
+      <li class="tabs-title <%= is_active?(goal) %>">
+        <%= link_to "#goal_#{goal.code}_targets_tab", "aria-selected": goal.code == 1 do %>
+          <% render SDG::Goals::IconComponent.new(goal) %>
+        <% end %>
+      </li>
+    <% end %>
+  </ul>
+
+  <div class="tabs-content" data-tabs-content="help_tabs">
+    <% @goals.each do |goal| %>
+      <div id="goal_<%= goal.code %>_targets_tab" class="tabs-panel tabs-icons <%= is_active?(goal) %>">
+        <h2><%= goal.code_and_title %></h2>
+        <%= sanitize t("sdg.goals.goal_#{goal.code}.long_description") %>
```

Comment:
How about using just `description` instead of `long_description`? People accessing the help page are usually in the middle of filling in a form, and IMHO the long description is too long and might distract users from the task they're doing.